.github/buildomat/jobs/xde.sh

```diff
@@ -80,4 +84,11 @@ cp $REL_SRC/xde $REL_TGT/
 sha256sum $REL_TGT/xde > $REL_TGT/xde.sha256
 
 cp $REL_LINK_SRC/libxde_link.so $REL_TGT/xde_link.so
-sha256sum $REL_TGT/xde_link.so > $REL_TGT/xde_link.so.sha256
\ No newline at end of file
+sha256sum $REL_TGT/xde_link.so > $REL_TGT/xde_link.so.sha256
+
+popd
+pushd tests
+cargo build --test loopback
+loopback_test=`find target/debug/deps -regex '.*loopback-[0-9a-f]*'`
```

Comment:
Probably not an issue in CI given it runs in a clean env, but if anyone runs this locally it might pick up an older artifact. Maybe something like this instead:

`cargo build --test loopback --message-format=json | jq -r "select(.profile.test == true) | .filenames[]"`